cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,12 @@
+/* global cy */
+
+describe('visit main page', () => {
+  it('make simple bigquery query', () => {
+    cy.visit('/')
+    cy.get('button:contains("Create Report")').click()
```

Comment:
can we use xpath or id to find a button? the text can be changed and tests will fail

Reply:
I see pros and cons: If the button has no text, for example, then the user will not recognize but test still pass